### PR TITLE
CCS Spectrometer C# Example of Absorption Measurement

### PR DESCRIPTION
**Pull request by ZiyingZhang313**
I've added the CCS spectrometer C# example of Absorption measurement. Could you please kindly review this and pull this if it works well? Thanks so much!

Best Regards
Zoe Zhang

### CODE REVIEW

File: C sharp/Thorlabs CCS Spectrometers/CCS - Absorption Measurement/CCS - Absorption Measurement/Program.cs [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+            title1.Text = "Absorption Spectrum";
+
+            Application.Run(form1);
+        }
+
+        /// <summary>
+        /// plot the optical density spectrum
+        /// </summary>
+        private void ChartOD(double[] DataIntensity, double[] DataWavelength)
```
Comment (manzelmi-thorlabs) [inline C sharp/Thorlabs CCS Spectrometers/CCS - Absorption Measurement/CCS - Absorption Measurement/Program.cs:220]: Can we change the name of this to ShowODChart?

File: C sharp/Thorlabs CCS Spectrometers/CCS - Absorption Measurement/CCS - Absorption Measurement/Program.cs [hunk 2]
```diff
@@ -0,0 +1,276 @@
+﻿using System;
```
Comment (manzelmi-thorlabs) [inline C sharp/Thorlabs CCS Spectrometers/CCS - Absorption Measurement/CCS - Absorption Measurement/Program.cs:1]: The code looks great. We might need to make a few changes to method names to make them more clear. We should also update the readme to include a description of the new example

**Review by manzelmi-thorlabs — CHANGES_REQUESTED** [review]

**Review by manzelmi-thorlabs — APPROVED** [review]
Looks  great!

**Review by wskopalik — APPROVED** [review]
Looks good!

**Pull request merged**